src/views/open-api-docs-view-v2/schema-transforms/schema-transform-shorten-hcp.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type { OpenAPIV3 } from 'openapi-types'
+import { schemaTransformTitle } from './schema-transform-title'
+
+/**
+ * Replaces "HashiCorp Cloud Platform" with "HCP" in the given string.
+ */
+function shortenHcp(s: string): string {
+	return s.replace('HashiCorp Cloud Platform', 'HCP')
+}
+
+/**
+ * Given an OpenAPI schema document,
+ * Return the document with the title modified, with any instances of
+ * "HashiCorp Cloud Platform" replaced with "HCP".
+ */
+export function schemaTransformShortenHcp(
```

Comment:
The broad intent with this `V2` version of our OpenAPI docs view is to first migrate all existing OpenAPI pages to it, then to delete the old `open-api-docs-view`, then rename this new one to remove the `V2` prefixes.

With this in mind, it felt like a good idea to copy over any utilities from within the old view folder. This way, when migration is complete, the old and new views shouldn't be entangled at all, and we should have an easier time deleting the old view folder.